src/components/ContributionInfo.vue

```diff
@@ -67,5 +73,24 @@ export default defineComponent({
 
 .markdown-card-text {
   padding: 16px;
+  text-align: start;
+}
+</style>
+
+<style>
```

Comment:
Couldn't get the embedded content to word with scoped style. I presume because the Markdown is rendered client side. If you can get it to apply while scoped, all for it.